scope.go

```diff
@@ -379,8 +380,7 @@ func (scope *Scope) ApplyToEvent(event *Event, hint *EventHint) *Event {
 		}
 	}
 
-	var emptyUser User
-	if event.User == emptyUser {
+	if reflect.DeepEqual(event.User, User{}) {
```

Comment:
If we remove `Data`, the fields should become comparable, so we no longer have to use `reflect.deepEqual`

Reply:
Yep, this was only added bec of the new Data map.